src/Microsoft.AspNetCore.Mvc/Microsoft.AspNetCore.Mvc.csproj

```diff
@@ -9,7 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\Microsoft.AspNetCore.Mvc.Analyzers\Microsoft.AspNetCore.Mvc.Analyzers.csproj" PrivateAssets="None" ReferenceOutputAssembly="false" />
+    <ProjectReference Include="..\Microsoft.AspNetCore.Mvc.Analyzers\Microsoft.AspNetCore.Mvc.Analyzers.csproj" PrivateAssets="contentfiles;build" ReferenceOutputAssembly="false" />
```

Comment:
what does this mean?

Reply:
Ahh, nevermind. I'll revert this - I was messing around to figure out a possible fix for https://github.com/aspnet/Mvc/issues/7864